examples/README.md

```diff
@@ -15,3 +15,82 @@ $ kubectl create -f pod.yaml
 Other helper scripts:
 * `logs.sh` output of the plugin
 * `exec-bash.sh` logs into the plugin's container and runs bash
+
+## How to test RBD Snapshot feature
+
+Before continuing, make sure you enabled the required feature gate 'VolumeSnapshotDataSource=true' in your Kubernetes cluster.
+
+In the `examples/rbd` directory you will find two files related to snapshots: [snapshotclass.yaml](./rbd/snapshotclass.yaml) and [snapshot.yaml](./rbd/snapshot.yaml).
+
+Once you created your RBD volume, you'll need to customize at least `snapshotclass.yaml` and make sure the `monitors` and `pool` parameters match your Ceph cluster setup. If you followed the documentation to create the rbdplugin, you shouldn't have to edit any other file.
+
+After configuring everything you needed, deploy the snapshot class:
+
+```bash
+$ kubectl create -f snapshotclass.yaml
+```
+
+Verify that the snapshot class was created:
+
+```bash
+$ kubectl get volumesnapshotclass
+NAME                      AGE
+csi-rbdplugin-snapclass   4s
+```
+
+Create a snapshot from the existing PVC:
+
+```bash
+$ kubectl create -f snapshot.yaml
```

Comment:
check https://github.com/ceph/ceph-csi/pull/174#discussion_r254152380, I think we need to remove `$` in case of no output

Reply:
Do we want this output?

volumesnapshotclass.snapshot.storage.k8s.io/csi-rbdplugin-snapclass created

Reply:
we have 2 options
```
$ kubectl create -f snapshot.yaml
volumesnapshotclass.snapshot.storage.k8s.io/csi-rbdplugin-snapclass created
```

or
```
kubectl create -f snapshot.yaml
```

note `$` is removed in second option

Reply:
OK, I removed the $ when there is no output.
Thanks for the review